### ISSUE TITLE
jalangi.analyze does not always perform an analysis

### ISSUE THREAD

**Issue by esbena** [Samsung/jalangi2#58]
## Description
It seems that `require('jalangi2').analyze(...)` can fail to perform an analysis by silently *not* loading the application of interest. 

When the failure occurs, it occurs on every instrument-and-analyze attempt for an application except for the first. The situation only occurs if the instrumented files are named the same as in the previous attempt.

## Abstract example
```
$ node instrument-and-analyze.js test1.js instumentationDirectory
<<instumentationDirectory/test1.js outputs>>

$ node instrument-and-analyze.js test1.js instumentationDirectory
<<instumentationDirectory/test1.js outputs>>

$ node instrument-and-analyze.js test2.js instumentationDirectory
<<instumentationDirectory/test2.js outputs>>

$ node instrument-and-analyze.js test2.js instumentationDirectory
<<instumentationDirectory/test2.js NOT LOADED>>
```
## Concrete example
See https://github.com/esbena/jalangi-api-caching-bug

## Additional information
 - reproducible with node 0.10 and 0.12 on ubuntu.
 - src/js/commands/{instrument,jalangi,direct}.js does not encounter the problem

**Comment by msridhar** [comment]
Reproduced; weird.  I'm looking into it.

**Comment by msridhar** [comment]
This was due to some horrible bug in ncp 2.0.0.  Reverting to 0.6.0 fixed the problem.  We should really fix #16 at some point.

BTW, @ksen007  this is exactly why I am against just updating library dependencies in `package.json` to the latest versions.

**Comment by msridhar** [comment]
@esbena I published jalangi2@0.2.3 in the npmjs database with this fix.
